geps/gep-2659.md

```diff
@@ -0,0 +1,199 @@
+# GEP-2659: Document and improve the GEP process
+
+* Issue: [#2659](https://github.com/kubernetes-sigs/gateway-api/issues/2659)
+* Type: Memorandum
+* Status: Accepted
```

Comment:
Would it be simpler to just have a `Memorandum` status instead of a separate type?

Reply:
Hmmm, maybe? I'll have a think about that.